etl/pad/raw/skills/leader_skill_info.py

```diff
@@ -1751,6 +1751,16 @@ def __init__(self, ms: MonsterSkill):
     def text(self, converter) -> str:
         return converter.color_combo_bonus_combo_text(self)
 
+class LSHeartCrossCombo(LeaderSkill):
```

Comment:
where is this used?

Reply:
Skilltype 209 is used in the third subskill of the new KoF Leona reincarnation. Its params are [2], which indicate +2 combos.